htdocs/core/class/html.form.class.php

```diff
@@ -7286,7 +7286,7 @@ public function selectProjects($selected = '', $htmlname = 'projectid', $filtert
 			$out .= $this->selectProjectsList($selected, $htmlname, $filtertype, $limit, $status, 0, $socid, $showempty, $forcecombo, $morecss);
 		}
 
-		if (empty($nooutput)) print $out;
```

Comment:
If nooutput is not set (0 = empty), we want output on screen, so we want print $out